css/interface.css

```diff
@@ -239,7 +240,10 @@ html {
 }
 
 .side-actions {
+  position: sticky;
+  top: 10px;
```

Comment:
why 10? maybe leave a comment explaining the reason

Reply:
> why 10? maybe leave a comment explaining the reason

Yes, there was nothing about top indentation in the task and it seemed to me that it'd be better when the block doesn't completely adhere to the site header. This is mostly related when a picture is selected. If it's not so good decision, then I'll remove it

Reply:
@tonytlwu can you jump in here to validate the code above and related comment?

@OleksandrLavrynenko I meant to leave a comment in the CSS so that it's clear why 10 and not 20, e.g.

```css
x {
  top: 10px; /* height of header */
}
```

Reply:
I'm fine with a `10px` padding.